OrbitGl/ManualInstrumentationManager.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_MANUAL_INSTRUMENTATION_MANAGER_H_
+#define ORBIT_GL_MANUAL_INSTRUMENTATION_MANAGER_H_
+
+#include "../Orbit.h"
+#include "OrbitBase/Logging.h"
+#include "capture_data.pb.h"
+
+class ManualInstrumentationManager {
```

Comment:
This class is basically unused, as it doesn't have fields, `OnTimerInfo` is empty and unused and `ApiEventFromTimerInfo` is static. What is the planned usage? Other `On...` methods are part of `CaptureListener`, implemented by `OrbitApp`, and add to `CaptureData`, but this doesn't seem to fit in it.

Reply:
Yes it's a bit weird in this PR, I should have put a note. I removed the unused method. The next PR populates the class properly.